test/controllers/sections/items/ItemProducerSizeControllerSpec.scala

```diff
@@ -159,7 +170,59 @@ class ItemProducerSizeControllerSpec extends SpecBase with MockPreDraftService w
         status(result) mustEqual SEE_OTHER
         redirectLocation(result).value mustEqual controllers.routes.JourneyRecoveryController.onPageLoad().url
       }
+    }
+
+    "for a GET unableToProvideInformation" - {
+
+      "must redirect to the next page" in new Setup(Some(
+        emptyUserAnswers.set(ItemExciseProductCodePage(0), testEpcWine)
+      )) {
+
+        MockUserAnswersService.set().returns(Future.successful(emptyUserAnswers))
```

Comment:
I think this should assert that the user answer for this page is removed when this link is clicked